### DISCUSSION TITLE
New demo release 11.11.0

### DISCUSSION THREAD

**Discussion by palexdev** [palexdev/MaterialFX#39] [Archived]
Binaries are in image\bin folder

Let me know what you think!

<hr /><em>This discussion was created from the release <a href='https://github.com/palexdev/MaterialFX/releases/tag/v11.11.0'>New demo release 11.11.0</a>.</em>

**Comment by NexRX** [comment]
Downloading the demo and will probably be refactoring some of my work (where needed) for this release! Good work getting it released!!!

**Reply by palexdev** [reply]
Thank you!! Have a nice refactor day haha 🙂

**Comment by mica-alex** [comment]
Demo looks great -- I don't see the 11.11.0 version on Maven yet, but I can't to get the update implemented as soon as it is!

**Reply by palexdev** [reply]
Yeah, Maven takes some time to update, it's still not listed, but you can already download it
